src/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+ 
+.card__image {
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+h1 {
+  margin: 0;
+}
+
+.card__title {
+  margin: 0 16px;
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 16px 16px;
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+p {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat ;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  background-position: center;
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
![Screenshot from 2023-05-17 13-07-41](https://github.com/mate-academy/layout_product-cards/assets/33029864/5a9213e2-8cf6-4668-839c-43f05508978b)

Reply:
<img width="554" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/05671e45-8a18-4c3f-9f2b-53350a74da67">
no, it does not work. reuse all styles from stars task